platform/linux-generic/include/odp/api/plat/atomic_inlines.h

```diff
@@ -82,6 +82,10 @@
 	#define odp_atomic_cas_acq_u32 __odp_atomic_cas_acq_u32
 	#define odp_atomic_cas_rel_u32 __odp_atomic_cas_rel_u32
 	#define odp_atomic_cas_acq_rel_u32 __odp_atomic_cas_acq_rel_u32
+	#define odp_atomic_cas_u128 __odp_atomic_cas_u128
```

Comment:
Please split api spec and implementation (and validation test) changes into separate patches. This part is implementation.

Reply:
I have split the commit into 2. 1 for spec and another 1 for implementation. Validation test will be checked in as separate commit.